Tools/GNUMake/Make.machines

```diff
@@ -127,3 +127,8 @@ ifneq (,$(filter $(host_name_env),poplar redwood tulip))
   which_computer := $(host_name_env)
 endif
 
+ifeq ($(findstring birch, $(host_name)),birch)
```

Comment:
Could we make the test to be that `host_name` contains both `birch` and `cray.com`?

Reply:
Yes, made a separate comment / suggestion in response since I couldn't come up with a readable one-line if statement to check both things.